context/app/static/js/components/Detail/Protocol/Protocol.jsx

```diff
@@ -26,12 +27,19 @@ function ProtocolLink(props) {
 function Protocol(props) {
   const { protocol_url } = props;
 
+  const lastWordPattern = new RegExp('([a-zA-Z0-9]+)$');
+  const matchedDoi = protocol_url.match(lastWordPattern)[0];
```

Comment:
```suggestion
  const matchedDoiSuffix = protocol_url.match(/\w+$/)[0];
```
- parens aren't needed if you're not extracting a substring.
- This is only part of a doi. (Splitting on slash might be safer - it could include dashes.)

Reply:
I agree.